flow-typed/platform.js

```diff
@@ -121,8 +121,9 @@ declare type QueryResponse = {
 
 declare type QueryFunction = (location: { [string]: mixed }) => Query;
 
-declare type AsyncPlatformRoute = {|
-	getComponent: () => Promise<React$ComponentType<*>>,
+declare type PlatformRoute = {|
```

Comment:
unwinding a separation I made earlier between an 'AsyncPlatformRoute' and a 'StaticPlatformRoute' - it was messing with types too much and is a little hard to handle with so many resolvers - I'll be able to restore it once `getIndexRoute` and `getNestedRoutes` are gone.